msticpy/data/drivers/resource_graph_driver.py

```diff
@@ -93,7 +93,7 @@ def connect(self, connection_str: str = None, **kwargs):
         )
         self.subscription_ids = [
             sub.subscription_id
-            for sub in self.sub_client.subscriptions.list()  # type: ignore
+            for sub in self.sub_client.subscriptions.list().append("")  # type: ignore
```

Comment:
I think this should be something like:
```python
self.subscription_ids = [
            sub.subscription_id
            for sub in self.sub_client.subscriptions.list()
]
self.subscription_ids.append("")
```
.append() adds an item to an existing list but doesn't return anything.
Also "" is not going to have a "subscription_id" attribute.